src/main/java/lotto/domain/AutoLottoGenerator.java

```diff
@@ -2,21 +2,30 @@
 
 import java.util.Collections;
 import java.util.List;
-import java.util.Set;
-import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.*;
+import static lotto.domain.LottoStore.ZERO;
 
 public class AutoLottoGenerator implements LottoGenerator {
+    private final int issueCount;
+
+    public AutoLottoGenerator(int issueCount) {
+        this.issueCount = issueCount;
+    }
 
     @Override
-    public Lotto generate() {
-        return Lotto.of(generateAutoNumbers());
+    public Lottos generate() {
+        return IntStream.range(ZERO, issueCount)
+                .mapToObj(lotto -> this.generateAutoNumbers())
+                .collect(collectingAndThen(toList(), Lottos::of));
```

Comment:
`
```suggestion
        return Stream.generate(this::generateAutoNumbers)
                .limit(issueCount)
                .collect(collectingAndThen(toList(), Lottos::of));

```

와 같이 생성하는것이 좀 더 명확하게 의미를 전달 할 수 있을 것 같아요.